packages/mdc-slider/test/component.test.ts

```diff
@@ -490,4 +490,29 @@ describe('MDCSlider', () => {
     document.body.removeChild(wrapper);
   });
 
+  it('correct value on pin value marker when value is 0', () => {
+    const root = getFixture();
+    Object.assign(root.style, {
+      position: 'absolute',
+      left: '0',
+      top: '0',
+      width: '400px',
+    });
+    root.classList.add(cssClasses.IS_DISCRETE);
+    root.setAttribute('aria-valuemin', '-100');
+    root.setAttribute('aria-valuemax', '100');
+    const component = new MDCSlider(root);
+    document.body.appendChild(root);
+    component.layout();
+
+    root.dispatchEvent(new MouseEvent('mousedown', {view: window, bubbles: true, cancelable: true, clientX: 199}));
+    root.dispatchEvent(new MouseEvent('mousemove', {view: window, bubbles: true, cancelable: true, clientX: 200}));
+    root.dispatchEvent(new MouseEvent('mouseup', {view: window, bubbles: true, cancelable: true, clientX: 200}));
+
+    const pinValueMarker = root.querySelector(strings.PIN_VALUE_MARKER_SELECTOR);
```

Comment:
Cast this `as HTMLElement` to fix a null check compilation error.

```ts
const pinValueMarker = root.querySelector(strings.PIN_VALUE_MARKER_SELECTOR) as HTMLElement;
```

Reply:
Done.